tests/benchmarks/test_h2o.py

```diff
@@ -31,8 +31,6 @@
         raise ValueError("Unknown h2o dataset(s): ", unknown_datasets)
 else:
     enabled_datasets = {
-        "0.5 GB (csv)",
```

Comment:
Do we have any other CSV tests? While I don't care deeply about this, I would care if this was 100% slower

Reply:
There's `test_csv.py::test_csv_basic`. If we want something more integrated, I'd rather include `5 GB (csv)`. `0.5 GB` just doesn't make any sense for `dask`.

Reply:
`test_csv_basic` works for me. I just would like to get notified if we're significantly slower all of a sudden